src/withAuthUserTokenSSR.js

```diff
@@ -8,6 +8,155 @@ import {
 import { getConfig } from 'src/config'
 import AuthAction from 'src/AuthAction'
 
+const determineAuthUser = async ({
```

Comment:
What are your thoughts on named async functions here instead of a const? Was the const a pattern you already saw within the library?

Also wondering if it would be good to maybe move some of these utils to their own file so that it keeps this file concise

Reply:
I was following the pattern I saw for the function syntax. The whole lib looks written without named functions.

I'll start working these other functions into their own file though 👍🏼